src/lib/jsonParseV2/parseRegistration.cpp

```diff
@@ -0,0 +1,367 @@
+/*
+*
+* Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Ken Zangelin
+*/
+#include <string>
+
+#include "rapidjson/document.h"
+
+#include "logMsg/logMsg.h"
+#include "common/string.h"
+#include "common/errorMessages.h"
+#include "common/globals.h"
+#include "rest/ConnectionInfo.h"
+#include "ngsi/Request.h"
+#include "apiTypesV2/Registration.h"
+#include "parse/forbiddenChars.h"
+#include "jsonParseV2/badInput.h"
+#include "jsonParseV2/utilsParse.h"
+#include "jsonParseV2/parseEntitiesVector.h"
+#include "jsonParseV2/parseStringVector.h"
+#include "jsonParseV2/jsonRequestTreat.h"
+
+
+
+#if 0
+------------------------------------------------------------------------------------
+  FIXME: Reference registration document (delete once POST /v2/registrations get implemented)
```

Comment:
Documentation stuff ...
To be included in this PR

Reply:
NTC? The comment is marked at outdated at github...

Reply:
Ah, yeah, I removed that if/endif

NTC